github/ci/prow/files/config.yaml

```diff
@@ -91,18 +91,32 @@ tide:
           skip-unknown-contexts: true
           from-branch-protection: true
 
-# we don't enable the branch protector itself yet, but want to
-# let tide know that travis-ci is needed
 branch-protection:
+#  protect: true
+  required_status_checks:
+    contexts:
+      - dco
```

Comment:
travis pushes directly. I don't think that we have any issues here, because it only pushes code from commits which are signed at this point already.

Reply:
but we can let kubevirt-bot sign any commit it does independently.

Reply:
makes sense!

Reply:
> Maybe even under org it won't work because of the `client-go` repo, which is updated by Travis only, which probably does not commit dco conform? Or can we disable dco on that repo only if it is enabled on the org? Or does `dco` only check PRs (Travis pushes tags directly to it without PR)? Sorry for the questions without answers, just thinking loud :)

The dco check will only get enabled if `protect: true` is set, which enabled the branch protection. This is currently not the case for the org, but only for a couple of branches. I'm currently writing a mail to repository owners asking them for what checks they need, so that we can centralize the configuration here.

We can disable the check for client-go by simply putting  `protect: false` under it. I guess I should add that directly.

Reply:
> We can disable the check for client-go by simply putting protect: false under it. I guess I should add that directly.
well, we want to have protection, only admins and travis should be able to push to master :)

Reply:
Now that you mention it: we could enforce PR approvals like this:

    required_pull_request_reviews:
      required_approving_review_count: 2

Reply:
on release branches, yes :+1: